src/analyzer/protocol/rawstream/events.bif

```diff
@@ -0,0 +1,27 @@
+## Generated for every DeliverStream() invocation on an instance of the Raw analyzer.
```

Comment:
Better not to phrase this in terms of internal methods, but instead describe directly what it does.